index.js

```diff
@@ -1,4 +1,40 @@
-export default function uniqueRandom(minimum, maximum) {
+function _exhaustive(minimum, maximum) {
+	function integersInRange(minimum, maximum) {
+		const array = [];
+		for (let index = minimum; index < maximum + 1; index++) {
+			array.push(index);
+		}
+
+		return array;
+	}
+
+	let restorer = [];
+	let toExtract = integersInRange(minimum, maximum);
+	let previousValue;
+
+	return function random() {
+		if (toExtract.length === 0) {
```

Comment:
Maybe run this check after the value is drawn so that it isn't run on the first run in which it would never trigger.

Reply:
Yes, I was thinking this too, but then will be a mess with `picker`, which should reassigned, since we're going to generate it first, then check if "refill" needed, refill and re-extract value...